__tests__/integration/one-app.spec.js

```diff
@@ -862,6 +862,32 @@ describe('Tests that require Docker setup', () => {
       });
     });
 
+    describe('custom error page', () => {
+      const loadCustomErrorPageRoot = async () => {
+        await addModuleToModuleMap({
+          moduleName: 'frank-lloyd-root',
+          version: '0.0.4',
```

Comment:
i don't think we needed a whole new frank for this. each new sample module version increases the cost of running integration tests. can you try to use an existing version ?

Reply:
@JAdshead we can, it was suggested to create a new one, but I can add it to an existing one if the team agrees.